gf-ui-lib/src/components/CheckboxField/CheckboxField.tsx

```diff
@@ -50,7 +50,8 @@ export const CheckboxField = (props: CheckboxFieldProps) => {
 
   return (
     <div className={styles.container}>
-      <SmallTitle
+      <Title
+        size="h4"
```

Comment:
Maybe better change naming from size to level? Because size associated with a font-size

Reply:
done